Apps/phgsgmail/gsgmail.json

```diff
@@ -111,13 +107,83 @@
             "description": "Contents of Service Account JSON file",
             "data_type": "password",
             "order": 1
+        },
+        "label": {
+            "description": "Mailbox Label (folder) to be polled",
```

Comment:
As this is an optional parameter, mention the behavior in the readme file or description in case of empty value